docs/goals.md

```diff
@@ -1,16 +1,60 @@
 # Goals and Non-Goals
 
-This document defines the purpose of the Message Format Working Group (MFWG)
-and informs the decisions about the scope and the priorities of its efforts.
+This document contains the charter for the MessageFormat Working Group (MFWG)
+and informs decisions about the scope and priority of its efforts.
 
-## Goals
+## Charter
 
-The primary task of the MFWG is to develop an industry standard for the
-representation of localizable dynamic message strings. A **_dynamic message
-string_** is a string whose content changes due to the value of or insertion
+A **_dynamic message string_** is a string whose content changes due to the value of or insertion
 of some data value or values.
 
-The design goals are listed below.
+The _Unicode MessageFormat Standard_ is an industry standard for the representation
+of localizable _dynamic message strings_.
+
+The MessageFormat Working Group maintains and extends the Unicode MessageFormat Standard,
+provides documentation;
+encourages implementation, including the development of tools and best practices;
+manages default and Unicode-defined function sets;
+and provides for interoperability with other standards.
+
+The MessageFormat Working Group is a working group of the CLDR-TC.
+
+## Goals
+
+- Maintain and develop the [messageformat.unicode.org] site,
+  maintaining a high bar (as a model for other Unicode documentation),
+  including at least 
+  a user guide,
+  and a migration guide from other formats (including ICU MessageFormat).
+- Encourage adoption of Unicode MessageFormat by:
+  providing implementation support materials;
+  creating tests;
+  linking to success stories, tools, and implementations;
+  supporting or hosting workshops or supporting presentations by members;
+  and supporting adoption by encapsulating standards (such as ECMA-402, MessageResource, etc.)
+- Support a determination there are at least ### additional interoperable implementations.
+- Support migration and adoption by moving all functions and options necessary to match the features of
+  ICU MessageFormat ("MF1") to Stable.
+- Support migration and adoption by expanding the default function set to support
+  additional use cases.
+- Develop a machine-readable function description format or syntax to support the needs of
+  implementations, including localization tools.
+
```

Comment:
As follow-up from discussions on last week's call, I think we should add the following:
```suggestion
- Define a standard vocabulary for expression attributes and message properties/metadata,
  to enable better interoperation between translation tools and platforms.
- Incubate and support working groups or interest groups
  that promote adoption of Unicode MessageFormat,
  such as the proposed working group to develop a standard message resource format,
  i.e. a new file format for bundles or collections of messages.

```